examples/keyed_list/src/main.rs

```diff
@@ -147,66 +154,95 @@ impl Component for Model {
+                            { "Prepend 5" }
+                        </button>
+                    </div>
                 </div>
-                <div class="ratio">
-                    <label for="ratio">{ "Person type ratio (0=only tags <= ratio <= 1=only components): " }</label>
-                    <input
-                        class="input" type="text" id="ratio"
+                <div class="row">
+                    <div class="col">
+                        <button class="btn_size alert alert-warning" onclick=self.link.callback(|_| Msg::ToggleKeyed)>
+                            { if self.keyed { "Disable keys" } else { "Enable keys" } }
+                        </button>
+                    </div>
+                    <div class="col">
+                        <button class="btn_size alert alert-info" onclick=self.link.callback(|_| Msg::SwapRandom)>
+                            { "Swap random" }
+                        </button>
+                    </div>
+                    <div class="col">
+                        <button class="btn_size alert alert-info" onclick=self.link.callback(|_| Msg::ReverseList)>
+                            { "Reverse list" }
+                        </button>
+                    </div>
+                    <div class="col">
+                        <button class="btn_size alert alert-info" onclick=self.link.callback(|_| Msg::SortById)>
+                            { "Sort by id" }
+                        </button>
+                    </div>
+                    <div class="col">
+                        <button class="btn_size alert alert-info" onclick=self.link.callback(|_| Msg::SortByName)>
+                            { "Sort by name" }
+                        </button>
+                    </div>
+                    <div class="col">
+                        <button class="btn_size alert alert-info" onclick=self.link.callback(|_| Msg::SortByAge)>
+                            { "Sort by age" }
+                        </button>
+                    </div>
+                    <div class="col">
+                        <button class="btn_size alert alert-info" onclick=self.link.callback(|_| Msg::SortByAddress)>
+                            { "Sort by address" }
+                        </button>
+                    </div>
+                </div>
+                    <p class="h5">{ "Person type ratio (0=only tags <= ratio <= 1=only components): " }{ self.build_component_ratio }</p>
+                    <input name="ratio" type="range" class="form-control-range" min="0.0" max="1.0" step="any"
                         value=self.build_component_ratio
                         oninput=self.link.callback(|e: InputData| Msg::ChangeRatio(e.value))
                     />
-                </div>
-                <p>{ "Number of persons: " }{ self.persons.len() }</p>
-                <p>{ "Ids: " }{ ids }</p>
+                <p class="h5">{ "Number of persons: " }{ self.persons.len() }</p>
```

Comment:
I guess this depends on which version of English you use.
```suggestion
                <p class="h5">{ "Number of people: " }{ self.persons.len() }</p>
```

Reply:
Not sure here, I leave it unchanged temporarily.